stubs/opencv-python/cv2/cv2.pyi

```diff
@@ -0,0 +1,4623 @@
+WND_PROP_AUTOSIZE: int
+WND_PROP_FULLSCREEN: int
+WND_PROP_OPENGL: int
+WND_PROP_TOPMOST: int
+WND_PROP_VISIBLE: int
+WND_PROP_VSYNC: int
+_INPUT_ARRAY_CUDA_GPU_MAT: int
+_INPUT_ARRAY_CUDA_HOST_MEM: int
+_INPUT_ARRAY_EXPR: int
+_INPUT_ARRAY_FIXED_SIZE: int
+_INPUT_ARRAY_FIXED_TYPE: int
+_INPUT_ARRAY_KIND_MASK: int
+_INPUT_ARRAY_KIND_SHIFT: int
+_INPUT_ARRAY_MAT: int
+_INPUT_ARRAY_MATX: int
+_INPUT_ARRAY_NONE: int
+_INPUT_ARRAY_OPENGL_BUFFER: int
+_INPUT_ARRAY_STD_ARRAY: int
+_INPUT_ARRAY_STD_ARRAY_MAT: int
+_INPUT_ARRAY_STD_BOOL_VECTOR: int
+_INPUT_ARRAY_STD_VECTOR: int
+_INPUT_ARRAY_STD_VECTOR_CUDA_GPU_MAT: int
+_INPUT_ARRAY_STD_VECTOR_MAT: int
+_INPUT_ARRAY_STD_VECTOR_UMAT: int
+_INPUT_ARRAY_STD_VECTOR_VECTOR: int
+_INPUT_ARRAY_UMAT: int
+_InputArray_CUDA_GPU_MAT: int
+_InputArray_CUDA_HOST_MEM: int
+_InputArray_EXPR: int
+_InputArray_FIXED_SIZE: int
+_InputArray_FIXED_TYPE: int
+_InputArray_KIND_MASK: int
+_InputArray_KIND_SHIFT: int
+_InputArray_MAT: int
+_InputArray_MATX: int
+_InputArray_NONE: int
+_InputArray_OPENGL_BUFFER: int
+_InputArray_STD_ARRAY: int
+_InputArray_STD_ARRAY_MAT: int
+_InputArray_STD_BOOL_VECTOR: int
+_InputArray_STD_VECTOR: int
+_InputArray_STD_VECTOR_CUDA_GPU_MAT: int
+_InputArray_STD_VECTOR_MAT: int
+_InputArray_STD_VECTOR_UMAT: int
+_InputArray_STD_VECTOR_VECTOR: int
+_InputArray_UMAT: int
+_OUTPUT_ARRAY_DEPTH_MASK_16F: int
+_OUTPUT_ARRAY_DEPTH_MASK_16S: int
+_OUTPUT_ARRAY_DEPTH_MASK_16U: int
+_OUTPUT_ARRAY_DEPTH_MASK_32F: int
+_OUTPUT_ARRAY_DEPTH_MASK_32S: int
+_OUTPUT_ARRAY_DEPTH_MASK_64F: int
+_OUTPUT_ARRAY_DEPTH_MASK_8S: int
+_OUTPUT_ARRAY_DEPTH_MASK_8U: int
+_OUTPUT_ARRAY_DEPTH_MASK_ALL: int
+_OUTPUT_ARRAY_DEPTH_MASK_ALL_16F: int
+_OUTPUT_ARRAY_DEPTH_MASK_ALL_BUT_8S: int
+_OUTPUT_ARRAY_DEPTH_MASK_FLT: int
+_OutputArray_DEPTH_MASK_16F: int
+_OutputArray_DEPTH_MASK_16S: int
+_OutputArray_DEPTH_MASK_16U: int
+_OutputArray_DEPTH_MASK_32F: int
+_OutputArray_DEPTH_MASK_32S: int
+_OutputArray_DEPTH_MASK_64F: int
+_OutputArray_DEPTH_MASK_8S: int
+_OutputArray_DEPTH_MASK_8U: int
+_OutputArray_DEPTH_MASK_ALL: int
+_OutputArray_DEPTH_MASK_ALL_16F: int
+_OutputArray_DEPTH_MASK_ALL_BUT_8S: int
+_OutputArray_DEPTH_MASK_FLT: int
+__UMAT_USAGE_FLAGS_32BIT: int
+
+class AKAZE(Feature2D):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def create(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getDefaultName(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getDescriptorChannels(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getDescriptorSize(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getDescriptorType(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getDiffusivity(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getNOctaveLayers(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getNOctaves(self, *args, **kwargs) -> Any: ...  # incomplete
+    def getThreshold(self, *args, **kwargs) -> Any: ...  # incomplete
```

Comment:
Should these return `Incomplete` if they're incomplete? (Same question for a lot of the other functions with the `# incomplete` comment.)

Reply:
I wrote it similar to how we'd write:
`def __getattr__(name: str) -> Any: ...  # incomplete` (taken from CONTRIBUTING.md)

Just that in this case we know that name of the attribute, and that it's a function. But have no idea about the arguments.

To me, it makes it more clear that that args are not actually `*args, **kwargs`

`def getThreshold(self, *args, **kwargs): ... # incomplete`
or
`def getThreshold(self, *args, **kwargs) -> Incomplete: ...`
could work as well (although we tend to not use `Incomplete` by itself as a return type)

Reply:
In that case, you can just omit the return types instead of writing "-> Any". (Any should generally mean "we've looked at this function but we can't express its return type in the type system".)

Reply:
Done!